### ISSUE TITLE
Add NFNets to keras.applications

### ISSUE THREAD

**Issue by marcusturewicz** [keras-team/keras#15229]
***Transferring from https://github.com/tensorflow/tensorflow/issues/47233 as requested***

**System information**

- TensorFlow version (you are using): 2.4.1
- Are you willing to contribute it (Yes/No): Yes, as much as possible but probably need help.

**Describe the feature and the current behavior/state.**

DeepMind recently introduced their new "Normalizer-Free" [NFNets](https://arxiv.org/abs/2102.06171), matching the test accuracy of EfficientNet-B7 on ImageNet while being up to 8.7x faster to train, and their largest models attain a new state-of-the-art top-1 accuracy of 86.5%. It seems fitting that these would be available in keras since they surpass EfficientNets which are already available.

**Will this change the current api? How?**

This would introduce new APIs i.e., `tensorflow.keras.applications.NFNetF0` through to `tensorflow.keras.applications.NFNetF6`.

**Who will benefit with this feature?**

Anyone doing image classification, including transfer learning and fine tuning; a large portion of the machine learning community!

**Any Other info.**

DeepMind's [implementation](https://github.com/deepmind/deepmind-research/tree/master/nfnets)
Other reference implementations (some incomplete):
- https://github.com/ypeleg/nfnets-keras
- https://github.com/rwightman/pytorch-image-models/blob/master/timm/models/nfnet.py
- https://github.com/vballoli/nfnets-pytorch

**Comment by ysyyork** [comment]
Any ETAs here?

**Comment by fchollet** [comment]
Hi there,

Yes, we'd welcome such a contribution. However, we have no availability to work on it, so you'd have to open a PR yourself (we will review the PR and guide you). Do you have an ImageNet checkpoint available for the model?

**Comment by ysyyork** [comment]
Got it, i'll see if i can make a PR on this. i have some components built in the past but just need to clean it up a bit

**Comment by innat** [comment]
@fchollet would you mind tagging **Contributions welcome** label on this?

**Comment by innat** [comment]
@ypeleg any scope to contribute here?

**Comment by SuryanarayanaY** [comment]
Hello, Thank you for reporting an issue.

We're currently in the process of migrating the new Keras 3 code base from keras-team/keras-core to keras-team/keras.
Consequently, This issue may not be relevant to the Keras 3 code base . After the migration is successfully completed, feel free to reopen this Issue at keras-team/keras if you believe it remains relevant to the Keras 3 code base. If instead this Issue is a bug or security issue in legacy tf.keras, you can instead report a new issue at [keras-team/tf-keras](https://github.com/keras-team/tf-keras/issues), which hosts the TensorFlow-only, legacy version of Keras.

To know more about Keras 3, please read https://keras.io/keras_core/announcement/
